### DISCUSSION TITLE
failed to compile demo on windows

### DISCUSSION THREAD

**Discussion by ActivePeter** [WasmEdge/WasmEdge#2028] [Q&A]
![image](https://user-images.githubusercontent.com/8100696/198884112-59457a8c-7327-404b-9bc1-10e7273450a2.png)
https://github.com/ActivePeter/pa_code_playground/tree/main/wasm/wasm_edge_test The project is at here.
The guide I am following is at here [https://wasmedge.org/book/en/sdk/c.html](https://wasmedge.org/book/en/sdk/c.html)

**Comment by q82419** [comment]
Hi,
Could you describe how did you install the WasmEdge on your Windows?
Thanks.

**Reply by ActivePeter** [reply]
I found the windows pkg install cmd will install the 0.10 version. And I found this issue has been fixed in 0.11 version.

**Reply by hydai** [reply]
Ah, sorry about that. The winget-pkgs is not maintained by us. I will need some time to figure out how to publish new release to winget-pkgs.
